org.civicrm.bootstrapcivihr/scss/directives/horizontal-scroll-shadows.scss

```diff
@@ -0,0 +1,40 @@
+.horizontal-scroll-shadows-master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.horizontal-scroll-shadows-wrapper {
+  overflow-x: auto;
+
+  &::before,
+  &::after {
+    box-shadow: 0 0 10px 1px rgba(48, 40, 40, 0.5);
+    content: '';
+    display: block;
+    height: calc(100% - 10px); // if height is 100% then the shadow will be cut off
+    opacity: 0; // by default the shadows are invisible
+    position: absolute;
+    top: 5px;
+    transition: opacity 0.2s ease-out;
+    width: 1px; // in order to make the shadow appear
+    z-index: 1; // in order to appear on top of the inset content
+  }
+
+  &::before {
+    left: -1px; // This hides the actual object leaving the shadow still visible
+  }
+
+  &::after {
+    right: -1px;
+  }
+
+  &.insetShadowLeft::before,
```

Comment:
what's going on with naming convention here? In addition of not following BEM, you're using both camelCase and kebab-case

Reply:
Done.